include/logging/log_core.h

```diff
@@ -219,13 +219,13 @@ extern "C" {
 				.source_id = _id			    \
 			};						    \
 									    \
-			if ((1 << _level) & LOG_FUNCTION_PREFIX_MASK) {	    \
+			if (BIT(_level) & LOG_FUNCTION_PREFIX_MASK) {	\
```

Comment:
should this line also have != 0 instead of implicit boolean conversion?

Reply:
yes, thanks for spotting it, will fix